server/plugin/command.go

```diff
@@ -367,7 +387,18 @@ func (p *Plugin) handleSubscribesAdd(_ *plugin.Context, args *model.CommandArgs,
 		msg += "\n\n**Warning:** You subscribed to a private repository. Anyone with access to this channel will be able to read the events getting posted here."
 	}
 
-	return msg
+	post := &model.Post{
+		ChannelId: args.ChannelId,
+		UserId:    p.BotUserID,
+		Message:   msg,
+	}
+
+	if _, err := p.API.CreatePost(post); err != nil {
```

Comment:
We generally want to name `*model.AppError`s a different name than `err` to avoid issues with concrete type errors vs `error` interfaces

```suggestion
	if _, appErr := p.API.CreatePost(post); err != nil {
```